### PR TITLE
Javascript javascript2 week3

### PR DESCRIPTION
**Pull request by faheem102**

### CODE REVIEW

File: javascript/javascript2/week3/index.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    <div id= "position"></div>
+
+    
+        <div class="container">
+            <h1>Please click to start the game:</h1>
+    
+            <form>
+    
+                <p><span id="result"></span></p>
+                <p><span id="error" style="color: red; font-weight: bolder;"></span></p>
```
Comment (dinythomas89) [inline javascript/javascript2/week3/index.html:22]: better use these styling in the css file

File: javascript/javascript2/week3/index.html [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+                <div>
+                    <input type="number" min="1" max="5" name="numberOfClicks" id="numberOfClicks">
+    
+                    <input type="button" name="startGame" onClick="noOfGames()" value="Start game" id="startGame"
+                        style="background-color:forestgreen; color:white;">
```
Comment (dinythomas89) [inline javascript/javascript2/week3/index.html:27]: same here

File: javascript/javascript2/week3/index.js [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+}, 2500)}
+logOutText();
+
+//!--Step2--!
+
+const delayString= (delay, stringToLog) =>{
+    let delayTime = delay * 1000
+
+    setTimeout(() => {
+        console.log (`${stringToLog}`)
```
Comment (dinythomas89) [inline javascript/javascript2/week3/index.js:15]: console.log (stringToLog) is fine here

File: javascript/javascript2/week3/index.js [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+//!--Step9--!
+let shouldTellFunnyJoke = true;
+const logFunnyJoke = () =>{
+    console.log("'Funny Joke': Hear about the new restaurant called Karma? There is no Menu: You get what you deserve.")
+}
+const logBadJoke = () =>{
+    console.log("'Bad Joke': You know what I saw today? Everything I looked at.")
+}
+function jokeCreator(shouldTellFunnyJoke, logFunnyJoke, logBadJoke){
+    if(shouldTellFunnyJoke ===true){
```
Comment (dinythomas89) [inline javascript/javascript2/week3/index.js:104]: or `if(shouldTellFunnyJoke){}` works the same

**Review by dinythomas89 — APPROVED** [review]
Its not a good practice to use inline styling.

**Comment by faheem102** [comment]
hi diny 
thank you for your feedback. i have now made all the suggested changes. please have a look and if it is fine then let me know. i will merge the pull request after your approval. thank you

**Pull request merged**